test/sampling.jl

```diff
@@ -91,6 +91,8 @@ test_rng_use(sample, 1:10, 10)
 
     @test samplepair(rng, [3, 4, 2, 6, 8]) === (3, 8)
     @test samplepair(rng, [1, 2])          === (1, 2)
+
+    @test extrema(samplepair(rng, UInt128(2))) == UInt128.((1, 2))
```

Comment:
The `UInt128` on the RHS has no effect to check the return type.
```suggestion
    @test extrema(samplepair(rng, UInt128(2))) == (1, 2)
    @test eltype(samplepair(rng, UInt128(2))) === UInt128
```